src/pages/industries/telcos-datacenters.jsx

```diff
@@ -0,0 +1,178 @@
+    "Cilium provides a comprehensive suite of capabilities that empower telcos and data center providers to transition seamlessly from traditional infrastructure to cloud native environments. As operators transition from IPv4 to IPv6, Cilium's NAT46/64 support provides a smooth pathway to ease this transition. Cilium's IPv6 support with BGP lets users advertise their IPv6 Pod CIDRs. Cilium's SRv6 L3VPN offers a scalable and flexible solution for interconnecting multiple sites while maintaining end-to-end network slicing and service isolation. Notably, Cilium supports SCTP, a protocol typically used by service providers and mobile operators. ",
+  ],
+  imageSrc: BulldozerBee,
+  imageAlt: 'ebeedex public speaking bee',
+};
+
+const bellTestimonial = {
+  logo: BellLogo,
+  description:
+    "Bell is Canada's largest telecommunications company, providing Mobile phone, TV, high speed and wireless Internet, and residential Home phone services.",
+  quotedText:
+    'With Cilium, Kubernetes Pods now have a single interface with only a default route. Nothing complex, no forwarding rules, no insanities of IPv6 static routes! And from a developer perspective, you just assign a VRF and the rest is really quite simple.',
+  withPerson: true,
+  name: 'Daniel Bernier',
+  role: 'Technical  Director, Bell Canada',
+  url: 'https://www.youtube.com/watch?v=vJaOKGWiyvU',
+};
+
+const bulletSection1 = {
+  heading:
+    'Simplify Integration Between Kubernetes Environments and Existing Network Infrastructure With CIlium’s BGP',
+  withImage: true,
+  paragraphs: [
+    'Telco and data center operators often have a blend of cloud native and legacy systems that require seamless communication. BGP, a common data center connectivity protocol is not natively integrated in Kubernetes.',
+    " With Cilium's native BGP support, operators can interconnect their legacy environment with a Cilium-managed environment, integrate seamlessly with Top of Rack devices, and automatically advertise pod CIDRs to BGP neighbors. Operators can significantly reduce CAPEX by leveraging Cilium's standalone load balancer to replace expensive legacy hardware load balancers.",
+  ],
+  imageSrc: TelcoImage1,
+  imageAlt: 'cilium bgp',
+};
+
+const bulletSection2 = {
+  heading: 'Future-proof Operations with Cilium’s dual stack IPv4/IPv6 and NAT46/64 Support',
+  paragraphs: [
+    "Cilium Dual-stack IPv4/IPv6 Networking allows each pod to be allocated IPv4 and IPv6 addresses, ensuring seamless communication with modern IPv6 systems and legacy IPv4 applications and services. For businesses in the process of transitioning from IPv4 to IPv6, Cilium's NAT46/64 support provides a pathway to ease this transition. Cilium's IPv6 support with BGP lets users advertise their IPv6 Pod CIDRs. Together, these features offer scalability and prepare businesses for a future where IPv6 will be the norm rather than the exception.",
```

Comment:
```suggestion
    "Cilium's dual-stack IPv4/IPv6 networking allows each pod to be allocated IPv4 and IPv6 addresses, ensuring seamless communication with modern IPv6 systems and legacy IPv4 applications and services. For businesses in the process of transitioning from IPv4 to IPv6, Cilium's NAT46/64 support provides a pathway to ease this transition. Cilium's IPv6 support with BGP lets users advertise their IPv6 Pod CIDRs. Together, these features offer scalability and prepare businesses for a future where IPv6 will be the norm rather than the exception.",
```